express-get-json/server.js

```diff
@@ -0,0 +1,36 @@
+import express from 'express';
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95,
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100,
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92,
+  },
+};
+const app = express();
+
+app.get('/api/grades', (req, res) => {
+  const gradeArray = [];
+  for (const grade in grades) {
+    gradeArray.push(grades[grade]);
+  }
+  res.json(gradeArray);
+});
+
+const port = 8080;
```

Comment:
Typically you would put constants like this at the top so they're easier to find if you want to change them.